library/board.py

```diff
@@ -19,40 +19,79 @@ def __init__(self) -> None:
         self.game_over = False
         self.turn_one = True
 
+    def check_board_victory(self) -> typing.Optional[str]:
+        """Determine if the entire board has been won, and by who."""
+        grid_states = np.array(
+            (self.check_subboard_victory(i) for i in "789456123"), dtype=np.str_
+        )
+        # This might have off ordering, TODO on testing it
```

Comment:
Make sure to test this :p

Reply:
That happens later, when we're able to set the actual items in the grids :P